src/controller/NotificationController.js

```diff
@@ -0,0 +1,116 @@
+import dotenv from 'dotenv';
+import NotificationUtil from '../utilities/Notification';
+
+dotenv.config();
+
+
+/**
+ * @description class for controlling notifications
+ */
+class NotificationController {
+  /**
+   *@description opts into email notification
+
+   * @param {Object} req - Http request
+   * @param {Object} res - Http response
+   *
+   * @returns {boolean} - true or false
+   */
+  static async optIntoEmailNotifications(req, res) {
+    const isUserEmailNotificationOn = await NotificationUtil
+      .isUserEmailNotificationOn(req.user);
+
+    if (isUserEmailNotificationOn) {
+      return res.status(400).json({
+        success: false,
+        message: 'You are already subscribed for email notifications'
+      });
+    }
+    await NotificationUtil
+      .updateNotificationStatus({ emailNotification: true }, req.user);
+    res.status(200).json({
+      success: true,
+      message: 'You have subscribed for email notifications'
+    });
+  }
+
+  /**
+   *@description opts out of email notifications
+
```

Comment:
I feel "unsubscribe for email notifications" is better for the description. see line 38.